src/Tests/No_UTF8_BOM.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.IO;
+using System.Text;
+using NServiceBus.MessageInterfaces.MessageMapper.Reflection;
+using NServiceBus.Newtonsoft.Json;
+using NUnit.Framework;
+
+[TestFixture]
+public class No_UTF8_BOM
+{
+
+    [Test]
+    public void Run()
```

Comment:
Let's rename this to `Should_not_emit_UTF8_BOM`

Reply:
Done.